source/guides/faq.md

```diff
@@ -49,6 +49,55 @@ See <https://github.com/nix-community/home-manager>
 
 ## NixOS
 
+### How to run non-nix executables?
+
+NixOS cannot run dynamically linked executables intended for generic linux environments out of the box.
+This is because, by design, it does not have a global library path, nor does it follow the traditional linux standard for where program files are stored on the filesystem.
+
+:::{note}
+The NixOS community uses the term "FHS" for traditional linux environments, which is short for "[Filesystem Hierarchy Standard](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html)".
+:::
+
+There are a few ways to resolve this mismatch in environment expectations:
+
+- Use the version packaged in nixpkgs, if there is one.
+  You can search available packages at <https://search.nixos.org/packages>.
+
+- Write a nix expression for the program to package it in your own configuration.
```

Comment:
```suggestion
- Write a Nix expression for the program to package it for your own configuration.

```

Reply:
I actually think "in" is better. Users unfamiliar with the nix language often don't realize that they can locate such an expression in or alongside their `configuration.nix`, and assume they need to have a local checkout of nixpkgs in order to have their own packages.

Reply:
Makes sense, but the intent is certainly not evident from just that word. Fixed in a force push.